pkg/minikube/cluster/cluster.go

```diff
@@ -305,7 +305,7 @@ func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 	if err := api.Create(h); err != nil {
 		// Wait for all the logs to reach the client
 		time.Sleep(2 * time.Second)
-		return nil, fmt.Errorf("Error creating. %s", err)
+		return nil, fmt.Errorf("Error creating store. %s", err)
```

Comment:
The best I can find is  _h.HostOptions.AuthOptions.StorePath_, which is not really about creating a _store_. If anything the VM is created here.